compute/acceptance/cluster_test.go

```diff
@@ -86,7 +96,7 @@ func TestAwsAccClusterResource_ValidatePlan(t *testing.T) {
 		"instance_profile_arn": "my_instance_profile_arn",
 	}
 	instancePoolLine := testGetClusterInstancePoolConfig("demo_instance_pool_id")
-	acceptance.AccTest(t, resource.TestCase{
+	resource.Test(t, resource.TestCase{
```

Comment:
`acceptance.AccTest` should be used instead of `resource.Test`, because it's setting "providers" map & makes `IsUnitTest` (debug mode for VScode launched with CLOUD_ENV)

Reply:
done